core/hybrisadaptor.cpp

```diff
@@ -249,23 +241,15 @@ bool HybrisManager::resumeReader(HybrisAdaptor *adaptor)
 
 void HybrisManager::standbyReader(HybrisAdaptor *adaptor)
 {
-    sensordLogD() << Q_FUNC_INFO  << adaptor->id() << adaptor->deviceStandbyOverride(); //alwaysOn
-    QList <HybrisAdaptor *> list;
-    list = registeredAdaptors.values();
-    bool okToStandby = true;
-    for (int i = 0; i < list.count(); i++) {
-        if (adaptor->sensorType  == list.at(i)->sensorType && list.at(i)->isRunning()) {
-            qDebug() << Q_FUNC_INFO << list.at(i) << "running";
-            okToStandby = false;
-        }
-    }
-
-    if (okToStandby) {
-        sensordLogD() << "deactivating for standby" << adaptor->name();
+    sensordLogD() << Q_FUNC_INFO << adaptor->id() << adaptor->deviceStandbyOverride();
+    if (!adaptor->deviceStandbyOverride()) {
+        setDelay(adaptor->sensorHandle, 1 * 1000000);
```

Comment:
I thought about doing this (and it's a good idea), but we should not presume that all sensors/clients will be happy about slowing down data when screen is blank.
I would rather set the delay for blanking somewhere else, like in the client.